src/components/BContainer/BContainer.test.js

```diff
@@ -1,44 +1,44 @@
-import { mount } from 'vue-test-utils'
-import BContainer from './BContainer'
+import { mount } from 'vue-test-utils';
+import BContainer from './BContainer';
 
 describe('BContainer', () => {
   it('renders correctly', () => {
-    const label = 'hello world'
-    const headerLevel = 1
+    const label = 'hello world';
+    const headerLevel = 1;
     const wrapper = mount(BContainer, {
-      propsData: { label, headerLevel }
-    })
-    expect(wrapper.html()).toMatchSnapshot()
-  })
-  it(`renders an element with no h element if hideLabel is passed`, () => {
```

Comment:
I would actually vote to use ``` everywhere.

Reply:
That is possible: https://eslint.org/docs/rules/quotes

I can change to that if we get a quorum.

Discuss.

Reply:
I'm good either way here